airbyte-config/config-models/src/main/resources/types/StandardDestinationDefinition.yaml

```diff
@@ -58,3 +58,10 @@ properties:
   protocolVersion:
     type: string
     description: the Airbyte Protocol version supported by the connector
+  normalizationRepository:
+    type: string
+  normalizationTag:
+    type: string
```

Comment:
Shouldn't this field be nullable or have a default value as they won't be set on all destinations.

Reply:
Yes you're right. All these fields can be NULL. Therefore, I did not add these fields to the required section (only in this case it was necessary to add some default value)

Reply:
> 

As for formatting - all files I added or changed accord to our format. But there are other files that are not related to my pull request - they were edited by the formatting team. Do I need to add them to my pull request as well? Previously, we rejected these changes and did not add them to our pull requests.

Reply:
Regarding the version, I was looking at the active version of the platform. At the time the class was created, it was version 0.4.15 and the migration was successfully deployed to the database after a new platform build.